frontend/javascripts/admin/dataset/dataset_add_zarr_view.tsx

```diff
@@ -311,7 +311,6 @@ function AddZarrLayer({
       (userInput.indexOf("https://") !== 0 && userInput.indexOf("s3://") === 0)
     ) {
       setSelectedProtocol(userInput.indexOf("https://") === 0 ? "https" : "s3");
-      setShowCredentialsFields(userInput.indexOf("s3://") === 0);
```

Comment:
note that all supported schemas now support their form of credentials so this is no longer needed